libs/payments/paypal/src/lib/util.ts

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { L_LIST } from './constants';
+import { IpnMerchPmtType, IpnMessage } from './types';
+
+/**
+ * Returns an ISO string without milliseconds appropriate for PayPal dates.
+ *
+ * @param d
```

Comment:
If we aren't adding documentation to the param, we shouldn't need to include it in the doc string, right?

Reply:
These docstrings are artifacts I missed from the migrated code. Removing!